lib/dynamoid/config.rb

```diff
@@ -36,6 +36,7 @@ module Config
     option :read_capacity, default: 100
     option :write_capacity, default: 20
     option :warn_on_scan, default: true
+    option :error_on_scan, default: false
```

Comment:
Could you please also update the README.md file and document this config option?

https://github.com/Dynamoid/dynamoid/blob/master/README.md?plain=1#L1274

Reply:
updated